test/pytest/test_multi_dense.py

```diff
@@ -11,46 +11,32 @@
 
 
 @pytest.mark.parametrize(
-    'backend, io_type',
+    'backend, strategy',
     [
-        ('Quartus', 'io_parallel'),
-        ('Vivado', 'io_parallel'),
-        ('Vitis', 'io_parallel'),
-        ('Vivado', 'io_stream'),
-        ('Vivado', 'io_stream'),
-        ('Vitis', 'io_stream'),
+        ('Vitis', 'Latency'),
```

Comment:
We don't need to test Vivado anymore?

Reply:
I wonder if I got ahead of myself. I was thinking to keep the tests shorter, since Vivado support will soon be on the way out, and it shares things with Vitis, but maybe it shouldn't be just yet. The tests aren't very long. I can add it.

Reply:
I was mostly curious if we are starting to phase out Vivado. Still catching up on the status and plans for things :) I guess it makes sense, so I don't think you necessarily need to add them.

Reply:
I think the plan was that after 1.0 we make Vitis the default backend and restructure the code a bit. I think it makes sense to add Vivado. (There's a similar Quartus -> oneAPI evolution that will happen.)